app/validation/validator.py

```diff
@@ -193,6 +195,29 @@ def validate_duplicates(self, json_to_validate, ignored_keys, special_key):
 
         return duplicate_errors
 
+    def validate_duplicate_options(self, json_to_validate):
+        errors = []
+
+        for block in self._get_blocks(json_to_validate):
+            answers_for_block = self._get_answers_for_block(block)
+
+            for answers in answers_for_block:
+                labels = set()
+                values = set()
+
+                for option in answers.get('options', []):
+
+                    if option['label'] in labels:
+                        errors.append(self._error_message('Duplicate label found - {}'.format(option['label'])))
+
+                    if option['value'] in values:
```

Comment:
What if we wanted 2 different option labels that have the same value in the background?
I think we have had this before in one of the surveys when it was split on the display to make the options more refined but downstream the value was the same.

Reply:
Noting for others. Having duplicate values will cause the functionality of the answer type, ie Radio buttons to not work properly. Two options with the same value mean the top option will always be chosen even if they chose the bottom option which has a different label but same value.